Platform/ElkhartlakeBoardPkg/CfgData/CfgData_Silicon.yaml

```diff
@@ -1857,21 +1857,43 @@
       option       : $EN_DIS
       help         : >
                      When FALSE, it disables Timed GPIO 0.
-      length       : 1b
+      length       : 0x01
       value        : 0x0
   - EnableTimedGpio1 :
       name         : Timed GPIO 1
       type         : Combo
       option       : $EN_DIS
       help         : >
                      When FALSE, it disables Timed GPIO 1.
-      length       : 1b
+      length       : 0x01
       value        : 0x0
+  - PchTsnGbeSgmiiEnable :
+      name         : PCH TSN SGMII Support
+      type         : Combo
+      option       : $EN_DIS
+      help         : >
+                     Enable/disable PCH TSN SGMII support 0=Disable 1=Enable
+      length       : 0x01
+      value        : 0x1
+  - PseTsnGbeSgmiiEnable :
+      name         : PSE TSN SGMII Support
+      type         : EditNum, HEX, (0x00,0xFFFF)
+      help         : >
+                     Enable/disable PSE TSN SGMII support 0=Disable 1=Enable
+      length       : 0x2
+      value        : { 0x00, 0x00 }
+  - PseTsnGbePhyInterfaceType :
+      name         : PSE TSN Phy Interface Type
+      type         : EditNum, HEX, (0x00,0xFFFF)
+      help         : >
+                     Set PSE TSN Phy Interface Type 0: Not Connected, 1: RGMII, 2: SGMII, 3:SGMII+
+      length       : 0x2
+      value        : { 0x01, 0x01 }
   - Dummy        :
       name         : SaPostMemTestRsvd
       type         : Combo
       option       : $EN_DIS
       help         : >
                      Reserved for SA Post-Mem Test
-      length       : 22b
+      length       : 0x4

```

Comment:
Why do you need to pad dummy 4 bytes?   
Dummy is only used if the structure is not 4 bytes aligned.  So the length should be 1 to 3.  Otherwise, dummy should be removed.